jenkinsci/tuleap-git-branch-source-plugin
# Add the Tuleap Git SCM  Browser

Part of [story#18320](https://tuleap.net/plugins/tracker/?aid=18320) native support of pull requests in tuleap branch source jenkins plugin

How to test:

 - In your Tuleap repository push new content which contains a new file,
   a deleted file and an edited file.
 - In Jenkins, launch a new build of the modified branch job
 - When the build is finish go into the job and click on the `details`
   link under the `Changes` section.
=> You should see the modified/removed and added file and commit id
=> Click on the different links and you should be redirected to the
concerned Tuleap file/commit/diff
=> When is removed, there is no link
